Documentation/github-connector.md

```diff
@@ -8,7 +8,7 @@ When a client redeems a refresh token through dex, dex will re-query GitHub to u
 
 ## Configuration
 
-Register a new application with [GitHub][github-oauth2] ensuring the callback URL is `(dex issuer)/callback`. For example if dex is listening at the non-root path `https://auth.example.com/dex` the callback would be `https://auth.example.com/dex/callback`.
+Register a new application with [GitHub][github-oauth2] ensuring the callback URL is `(dex issuer)/callback`. For example if dex is listening at the non-root path `https://auth.example.com/dex` the callback would be `https://auth.example.com/dex/callback`. Please note that a user needs to mark their email id as public in GitHub, so that the ID token that gets returned for this user contains an email id field in the ID token claims.
```

Comment:
I think detail the fact that GitHub's API won't return an email if the end user's email isn't public. However you want to word that, but the current sentence isn't super clear.